.circleci/download_reference_lhes.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
```

Comment:
I comment here, but it applies everywhere else. Currently, our JHUGen repository is still public, and this is why downloading references might still work. However, once this repository goes public, this will likely no longer be the case. I suggest moving these .lhe files into their own public repository and download them from there. This way, we don't also have to make additional commits to this repository instead of simply rerunning the CircleCi checks once more.

Reply:
I would also remove the .lhe files from the history of this PR so that checkout doesn't have to go through them.